doc/source/data/getting-started.rst

```diff
@@ -151,6 +133,11 @@ since your map function can return an output batch of any size.
     ds.take(5)
     # -> [{'value': 0}, {'value': 2}, ...]
 
+The batch format can be specified using ``batch_format`` option, which defaults to "native",
+meaning pandas format for Arrow-compatible batches, and Python lists for other types. You
+can also specify explicitly "arrow" or "pandas" to force a conversion to that batch format.
+The batch size can also be chosen. If not given, the batch size will default to entire blocks.
+
 By default, transformations are executed using Ray tasks.
 For transformations that require setup, specify ``compute=ray.data.ActorPoolStrategy(min, max)`` and Ray will use an autoscaling actor pool of ``min`` to ``max`` actors to execute your transforms.
 For a fixed-size actor pool, specify ``ActorPoolStrategy(n, n)``.
```

Comment:
Since we present map_batches() only, there is a usage of map() (line#163) for preprocessing which may be changed as well.